sample/build.gradle

```diff
@@ -122,3 +111,31 @@ android {
         }
     }
 }
+
+task syncPhotos(type: com.novoda.gradle.command.Files) {
+    deviceId {
+        def moto = android.command.devices().grep({ it.brand() == 'motorola' })
+        if (!moto) {
+            throw new GroovyRuntimeException('No Motorola device found')
+        }
+        moto[0].id
+    }
+    script {
+        def dir = '/sdcard/DCIM/Camera/'
+        def imageDir = file('motoPhoto')
```

Comment:
I would prefer having more descriptive names like `backupDir` and `deviceImageDir`, or something of the likes.